testing/bdd/templates/timeBase_timeformat.xml

```diff
@@ -47,7 +50,22 @@
   {% endif %}
 {% endif %}
     >
-    <tt:div>
+    <tt:div
+{% if div_begin %}
+  {% if div_begin == "*?Empty?*" %}
```

Comment:
Magic keyword? Think I've seen this before somewhere. Is this Gil's edit to allow pass-through of an empty value? Or does the `*?Empty?*` text need to appear actually within the scenario example? (it isn't in any of them so far)